cmd/mattermost/commands/webhook.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/spf13/cobra"
+)
+
+var WebhookCmd = &cobra.Command{
+	Use:   "webhook",
+	Short: "Management of webhooks",
+}
+
+var WebhookListCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List webhooks",
+	Long:    "list all webhooks",
+	Example: "  webhook list myteam",
+	RunE:    listWebhookCmdF,
+}
+
+func listWebhookCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	var teams []*model.Team
+	if len(args) < 1 {
+		// If no team is specified, list all teams
+		teams, err = app.GetAllTeams()
```

Comment:
@lologarithm 
Here short hand declaration should be there else it would panic in case of <nil> error.
Because error  returned by `InitDBCommandContextCobra` is of different type which is captured in `err` variable earlier.
Now we are trying to assign error from `app.GetAllTeams` to the same err variable but the type error returned here is different.
cc @jespino

Reply:
Very good catch! the firs one is an `error` and the second one is a `*model.AppError`. I think the simplest solution is tu define a var there like `var err *modelAppError`, and then keep the rest equal.

Reply:
`func InitDBCommandContextCobra(command *cobra.Command) (*app.App, error)` shows it returns the error type. Shouldn't cause any issues to reuse it (the unit test exercises this code path).

If you would prefer I create another error variable just to make the code clear this is separate I can do that.

Reply:
Yes, I prefer to have a new variable there. We have had problems with nil comparisons with different error types (we want to cleanup that in the future, but for now is better to have different variables).